cypress/integration/parallel-3/holidayStops.spec.ts

```diff
@@ -186,12 +186,13 @@ describe('Holiday stops', () => {
 
 		cy.findByRole('button', { name: 'Yes' }).click();
 		cy.wait('@delete_holiday_stop');
+		cy.wait('@product_detail');
 		cy.wait('@fetch_existing_holidays');
 
 		cy.get('table').contains('Deleted on 7 April 2022');
 
 		cy.get('@fetch_existing_holidays.all').should('have.length', 2);
-		cy.get('@product_detail.all').should('have.length', 1);
+		cy.get('@product_detail.all').should('have.length', 2);
```

Comment:
Why has this increased to 2?

Reply:
we are fetching all product details after deleting a stop because the page is doing a hard reload instead of the previous refetch logic